<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSICheck</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="checkProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Make only check procedure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ChooseLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chooseLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose device:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logTextBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>300</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check and repair log:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="chooseToolButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
